src/coreclr/tools/aot/DependencyGraphViewer/DependencyGraphsForm.cs

```diff
@@ -5,20 +5,31 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Data;
+using System.Diagnostics;
 using System.Drawing;
+using System.IO;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using Microsoft.Diagnostics.Tracing.Session;
 
 namespace DependencyLogViewer
 {
     public partial class DependencyGraphs : Form
     {
-        public DependencyGraphs()
+        private int _fileCount = -1;
+
+#nullable enable
+        public DependencyGraphs(string? argPath)
         {
             InitializeComponent();
+            if (argPath != null)
```

Comment:
Nit: Feels weird to just enable nullable here, I would get rid of the nullable enable here, have string and then check
```suggestion
            if (!string.IsNullOrEmpty(argPath))
```